src/cocaine-app/yt_worker.py

```diff
@@ -0,0 +1,176 @@
+from logging import getLogger
+from yql.api.v1.client import YqlClient
+import time
+import datetime
+
+
+logger = getLogger('mm.planner')
+
+
+class YqlWrapper(object):
+    def __init__(self, cluster, token, attempts, delay):
+        self._cluster = cluster
+        self._token = token
+        self._attempts = attempts
+        self._delay = delay
+        self.aggregate_query = """
```

Comment:
Done in https://github.com/yandex/mastermind/pull/311/commits/4732106cd00d7b48fb9a6db30b7cff4aef8db6df